src/languages/en.js

```diff
@@ -124,6 +124,7 @@ export default {
         areYouSure: 'Are you sure?',
         zipCodeExample: 'e.g. 12345, 12345-1234, 12345 1234',
         websiteExample: 'e.g. https://www.expensify.com',
+        format: ({zipSampleFormat}) => (zipSampleFormat ? `Format: ${zipSampleFormat}` : ''),
```

Comment:
This is currently not consistent with `format` in the `es.js` file - did I miss a discussion why we do this? I think we don't necessarily need to do this conditional check in the translation file, we can probably do it in the files that use this translation 🤔

Reply:
Niiice thanks for finding that @Prince-Mendiratta - I don't have a very strong opinion here so let's keep it as you have it now 👍